griptape/drivers/prompt/openai_chat_prompt_driver.py

```diff
@@ -189,6 +189,9 @@ def __to_openai_messages(self, messages: list[Message]) -> list[dict]:
                         ]
                     ],
                 }
+                # Some OpenAi-compatible services don't accept an empty array for content
+                if not openai_message["content"]:
+                    openai_message["content"] = ""
```

Comment:
we know for sure this doesnt get messed up with real OpenAI?

Reply:
Yes, tested.